### PR TITLE
Fix travis issues and refactor erlang install to support current kerl version

### PR DESCRIPTION
**Pull request by rhass**
This pulls in the PR #36 as well as fixes various travis issues around build timeouts. Additionally, the erlang install was refactored to support the current version of the `kerl`.

### CODE REVIEW

File: libraries/erlang_install.rb [hunk 1]
```diff
@@ -81,8 +82,12 @@ def install_dependencies
     # @see Chef::Resource::LanguageInstall#install
     #
     def install
-      build_erlang
-      install_erlang
+      if path_empty?(new_resource.prefix)
+        build_erlang
+        install_erlang
+      else
+        raise "Invalid prefix #{new_resource.prefix}. The prefix must empty or not exist."
```
Comment (yzl) [inline libraries/erlang_install.rb]: I would reword this to something like "The prefix #{new_resource.prefix} either does not exist or is empty."

Comment (rhass) [inline libraries/erlang_install.rb]: :+1: Thanks!

**Comment by schisamo** [comment]
Thanks for getting this all fixed up @rhass. :shipit:
